src/lib/clusters/clusters.tsx

```diff
@@ -589,10 +587,8 @@ async function kubeconfigAction({
           name: item.username || "",
           token: decryptedToken || "",
         });
-      } catch (err) {
-        console.error(
-          `Failed to decrypt token: ${err}, ${credential.username}`,
-        );
+      } catch (_err) {
+        // Silently continue on decryption errors
```

Comment:
Could you create linear and add ID here and above?

Reply:
[7e516a4](https://github.com/sfcompute/cli/pull/142/commits/7e516a4c48c743a2b85254dd950bf5e3e9daa157)